### PR TITLE
removed go-ethereum from dependencies

### PR DESCRIPTION
**Pull request by evgensheff**

### CODE REVIEW

File: db/substate_db.go [hunk 1]
```diff
@@ -24,6 +25,8 @@ type SubstateDB interface {
 	// GetSubstate gets the Substate for given block and tx number.
 	GetSubstate(block uint64, tx int) (*substate.Substate, error)
 
+	GetBlockSubstates(block uint64) map[int]*substate.Substate
```
Comment (petr-hanzl) [inline db/substate_db.go:28]: nit: Could you add documentation what does this function do please?

File: db/substate_db.go [hunk 2]
```diff
@@ -88,6 +91,48 @@ func (db *substateDB) GetSubstate(block uint64, tx int) (*substate.Substate, err
...
+			panic(fmt.Errorf("cannot decode data into substate: %v", err))
+		}
+
+		txSubstate[tx] = sbstt
+	}
+	iter.Release()
+	err = iter.Error()
+	if err != nil {
+		panic(err)
+	}
```
Comment (petr-hanzl) [inline db/substate_db.go:131]: Please return the errors instead of `panicking`

**Review by petr-hanzl — CHANGES_REQUESTED** [review]

File: db/substate_db.go (hunk 1 above)
Comment (evgensheff) [reply db/substate_db.go]: Done

File: db/substate_db.go (hunk 2 above)
Comment (evgensheff) [reply db/substate_db.go]: Make sense, thanks

**Review by petr-hanzl — APPROVED** [review]

**Pull request merged**